docs/blog/posts/stable-release-announcement.md

```diff
@@ -0,0 +1,73 @@
+---
+date: 2024-03-10
+authors: [agardnerIT]
+description: >
+  In this blog post you will learn how to configure Keptn via the new Helm umbrella chart.
```

Comment:
I guess the description is a copy paste error, no?

Reply:
from Keptn umbrella charts blogpost

Reply:
good catch, fixed